### ISSUE TITLE
question

### ISSUE THREAD

**Issue by publicocean0** [guooscar/JazminServer#31]
i m searching a webrtc server for connecting my browser client directly to server 
i saw in readme they told about webrtc but i cant found sources about it

**Comment by publicocean0** [comment]
really i found it https://github.com/guooscar/JazminServer/blob/master/JazminServer/test/jazmin/test/server/relay/RelayServerDtlsTest.java but i have to understand if it is possible to extract media from that stream

**Comment by guooscar** [comment]
you need to understand webrtc spec . see https://webrtc.org/quick-links/

**Comment by publicocean0** [comment]
你好 我是 Cristian . I m testing my initial lessons in chinese :) :) 

Yes many documents to read and a lot of time for implementing. 
I found a library opensource java ... it seams permitting connections.

Anyway i gave a look to your project. It is a wonderfull project. Compliments. It is very complex solution.You are very good java expert.
Maybe i will convert it in maven with most recent tools.

**Comment by guooscar** [comment]
all webrtc files is under jazmin/server/relay/udp/webrtc package .
the DtlsRelayChannel and DtlsRelayChannelHandler is netty network processer 
if u want clean code about webrtc  that's all  what u need

**Comment by publicocean0** [comment]
i will gave a look :) .... i saw there is also a war. It is possible deploy a war using deployer?

**Comment by guooscar** [comment]
JazminServer is a standalone server program  not war.

**Comment by publicocean0** [comment]
yes , i was telling about   JazminDeployer ... it seams a war to deploy in jasminserver

**Comment by guooscar** [comment]
yes . JazminDeployer is just same like jenkins ,devops tool
